### PR TITLE
chore(webpack): switch to webpack 2

### PR DESCRIPTION
**Pull request by frassinier**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/Talend/bootstrap-theme/blob/master/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
Webpack 1 is used to run static server while contributing

**What is the new behavior?**
Switch to Webpack 2

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://webpack.js.org/guides/migrating/

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,104 +1,89 @@
+const webpack = require('webpack');
+const path = require('path');
 const autoprefixer = require('autoprefixer');
 
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
```
Comment (talend-glorieux) [inline webpack.config.js]: Why are you using double quotes?

Comment (frassinier) [inline webpack.config.js]: Oups! :astonished:

File: example/index.html [hunk 2]
```diff
@@ -5,7 +5,6 @@
     <title>Talend Bootstrap Theme example</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <script src="https://use.fontawesome.com/88247cfe1e.js"></script>
```
Comment (jmfrancois) [inline example/index.html]: this change has nothing to do here AFAIK

Comment (frassinier) [inline example/index.html]: Right, do you need another PR for that? (It's not used)
